spec/traits.dd

```diff
@@ -1330,6 +1330,31 @@ void main()
 ---
 )
 
+$(H3 $(GNAME getModuleClasses))
+
+        $(P Optionally takes one argument which is a module name or a fully qualified import.
```

Comment:
What happens if you do `import a = std.stdio;`? Is `a` considered fully qualified at that point?

What if you use it on an imported symbol instead of an imported module?

It also isn't really a module name per se, but rather a module symbol, since it isn't quoted.